src/scicat_configuration.py

```diff
@@ -69,7 +68,7 @@ def build_arg_parser(
 
     **Note**: It can't parse the annotations from parent class.
     """
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(argument_default=argparse.SUPPRESS)
```

Comment:
This option was missing...

Reply:
Good that we find the issue and we... you fixed it! :) 
Thank you for the quick fix